src/NetworkChest.lua

```diff
@@ -276,6 +282,16 @@ function M.on_entity_settings_pasted(event)
   end
 end
 
+function M.trash_to_network(trash_inv)
+  if trash_inv ~= nil then
+    for name, count in pairs(trash_inv.get_contents()) do
+      -- FIXME: check global network item limits? (no?)
```

Comment:
There aren't any trash limits for the player's logistics so no need to have limits for spidertrons.